### PR TITLE
[docs] update ulimit in macOS Quick Start

### PR DESCRIPTION
**Pull request by gargsans-yb**
Summary: Changes made in the quick-start/install/macos/ to include maxproc ulimit as a part of pre-requisites. Recommended value in 2500.

### CODE REVIEW

File: docs/content/latest/quick-start/install/macos.md [hunk 1]
```diff
@@ -76,10 +76,15 @@ showAsideToc: true
...
     ```sh
-    $ launchctl limit maxfiles
+    $ launchctl limit
     ```
 
-    We recommend setting the soft and hard limits to 1048576.
+    We recommend the following soft and hard limits.
+
+    ```output
+    maxproc     2500        2500
```
Comment (polarweasel) [inline docs/content/latest/quick-start/install/macos.md:85]: My machine has a maxproc value higher than this (2666). Are you sure you want me to lower it?

File: docs/content/latest/quick-start/install/macos.md [hunk 2]
```diff
@@ -90,7 +95,7 @@ showAsideToc: true
     kern.maxfilesperproc=1048576
     ```
 
-    If this file does not exist, then create the file `/Library/LaunchDaemons/limit.maxfiles.plist` and insert the following:
+    If this file does not exist, then for setting maxfiles soft and hard limits, create the file `/Library/LaunchDaemons/limit.maxfiles.plist` and insert the following:
```
Comment (polarweasel) [inline docs/content/latest/quick-start/install/macos.md:98]:
```suggestion
    If this file does not exist, create the following two files:

    \
    Create `/Library/LaunchDaemons/limit.maxfiles.plist` and insert the following:
```

File: docs/content/latest/quick-start/install/macos.md [hunk 3]
```diff
@@ -115,10 +120,36 @@ showAsideToc: true
       </plist>
     ```
 
-    Ensure that the `plist` file is owned by `root:wheel` and has permissions `-rw-r--r--`. To take effect, you need to reboot your computer or run this command:
+    For setting maxproc soft and hard limits, create the file `/Library/LaunchDaemons/limit.maxproc.plist` and insert the following:
```
Comment (polarweasel) [inline docs/content/latest/quick-start/install/macos.md:123]:
```suggestion
    \
    Create `/Library/LaunchDaemons/limit.maxproc.plist` and insert the following:
```

File: docs/content/latest/quick-start/install/macos.md [hunk 4]
```diff
@@ -115,10 +120,36 @@ showAsideToc: true
...
+            </array>
+          <key>RunAtLoad</key>
+            <true/>
+          <key>ServiceIPC</key>
+            <false/>
+        </dict>
+      </plist>
+    ```
+
+    Ensure that the `plist` files are owned by `root:wheel` and has permissions `-rw-r--r--`. To take effect, you need to reboot your computer or run these command:
```
Comment (polarweasel) [inline docs/content/latest/quick-start/install/macos.md:148]:
```suggestion
    \
    Ensure that the `plist` files are owned by `root:wheel` and have permissions `-rw-r--r--`. To take effect, you need to reboot your computer or run the following commands:
```

**Review by polarweasel — CHANGES_REQUESTED** [review]
Minor changes...

File: docs/content/latest/quick-start/install/macos.md (hunk 1 above)
Comment (gargsans-yb) [reply docs/content/latest/quick-start/install/macos.md:85]: Changed the check to have at least minimum values

**Review by polarweasel — APPROVED** [review]
This is working right, now. @gargsans-yb please sign the CLA and then we're good to merge.

**Review by polarweasel — APPROVED** [review]

**Pull request merged**